crates/pipeline_manager/src/config.rs

```diff
@@ -237,25 +237,21 @@ impl ManagerConfig {
         Path::new(&self.working_directory).join("cargo_workspace")
     }
 
+    /// Where Postgres embed stores the database.
+    pub(crate) fn postgres_embed_data_dir(&self) -> PathBuf {
+        Path::new(&self.working_directory).join("data")
```

Comment:
Better to create a temporary folder. Adding folders to the source repo bloats the docker context.

Reply:
If we're running local then you'd want to keep the data around, no? 
(e.g., if you restart the manager it will find the db in data/ (instead of sqlite which kept manager.db in the working dir)

Reply:
I see. So it'll be in ~/.dbsp or something by default. Does the data get wiped in between tests?

Reply:
@lalithsuresh , I think "current working directory" is the DBSP working directory, usually `~/.dbsp`, not the Unix `cwd`.

Reply:
And I imagine that for test runs the DBSP working directory could be under `/tmp`.

Reply:
Yes

Test setup is slightly different: see here https://github.com/vmware/database-stream-processor/pull/375/files#diff-66137f70e107475f6293e863373cc39a0dcf6bd92443def5ed6c08d377e15f33R43

Reply:
well it's test.rs line 43 in case this link doesn't work